packaging/format-docs.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# Copyright 2007-2022 The OpenRA Developers (see AUTHORS)
+# This file is part of OpenRA, which is free software. It is made
+# available to you under the terms of the GNU General Public License
+# as published by the Free Software Foundation, either version 3 of
+# the License, or (at your option) any later version. For more
+# information, see COPYING.
+
+import io
+import sys
+import json
+from collections import OrderedDict
+
+def format_type_name(typeName):
+    name = typeName
+    if name.endswith("Info"):
+        name = name[0:-4]
+
+    return f'[`{name}`](#{name.lower()})'
+
+def format_docs(version, collectionName, types):
+    typesByNamespace = OrderedDict()
+    for currentType in types:
+        if currentType["Namespace"] in typesByNamespace:
+            typesByNamespace[currentType["Namespace"]].append(currentType)
+        else:
+            typesByNamespace[currentType["Namespace"]] = [currentType]
+
+    explanation = ""
+    if collectionName == "TraitInfos":
+        explanation = "all traits with their properties and their default values plus developer commentary"
+    elif collectionName == "WeaponTypes":
+        explanation = "a template for weapon definitions as well as its contained types (warheads and projectiles) with default values and developer commentary"
```

Comment:
Perhaps move `are` up to `explanations` so this part could use `is`

Reply:
But the current result is
> **"Listed below are a template for weapon definitions as well as its contained types (warheads and projectiles)"**

which I will probably change to
> **"Listed below are a template for weapon definitions and the types it can use (warheads and projectiles)"**

The `are` here sounds fine to me, as there are 2 things - the template and the list of types 🤔